app/src/main/res/values/strings.xml

```diff
@@ -58,6 +58,7 @@
     <string name="about_license">License</string>
     <string name="about_questionnaire">Questionnaire</string>
     <string name="about_version_prefix">"V%1$s</string>
+    <string name="about_inquiry_url">https://docs.google.com/forms/d/1fpxZOqprZyzBgMBb7HCCrq538oOzfzWkOMpItRqF-ro/viewform</string>
```

Comment:
I am afraid the form in English might have a different URL.  In such case, we can switch the URL based on the user's language if it is in string resource.